devicon.json

```diff
@@ -2535,6 +2535,31 @@
           }
         ]
     },
+    {
+        "name": "sqlalchemy",
+        "tags": [
+            "python",
+            "orm"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```

Comment:
You don't have to put the "plain-wordmark" here. Since there's no "sqlalchemy-plain-wordmark.svg", this would go into the `aliases` attribute